### DISCUSSION TITLE
How to stop ruff fighting with the default PyCharm formatter?

### DISCUSSION THREAD

**Discussion by meoyawn** [InSyncWithFoo/ryecharm#38] [Q&A]
I can't find a way to disable default PyCharm formatter except for unbinding the reformat key, in which case Ruff isn't executed as well, can you help?

**Comment by InSyncWithFoo** [comment]
For `.py`/`.pyi`/`.pyw` files, Ruff should override the default formatter entirely. Could you show a screenshot/screencast demonstrating the problem?
